internal/pkg/api/handleAck_test.go

```diff
@@ -101,15 +101,6 @@ func TestEventToActionResult(t *testing.T) {
 	}
 }
 
-// Mock bulker
-type mockBulk struct {
-	ftesting.MockBulk
-	searchErr error
-	createErr error
-	updateErr error
-	actions   []model.Action
-}
```

Comment:
I changed `ftesting.MockBulk` to use `testify/mock.Mock` this allows us to remove the multiple wrappers around it in different tests, and lets us define exactly what calls we expect when interacting with `bulk.Bulk` in tests